src/main/resources/lang/default.lang

```diff
@@ -2235,7 +2237,40 @@ attribute types:
 	player.sneaking_speed: player sneaking speed, sneaking speed
 	player.submerged_mining_speed: player submerged mining speed, submerged mining speed
 	player.sweeping_damage_ratio: player sweeping damage ratio, sweeping damage ratio
-	zombie.spawn_reinforcements: zombie spawn reinforcements
+	zombie.spawn_reinforcements: zombie spawn reinforcementsgeneric.armor: generic armor, armor
+	# 1.21.3 removes prefixes, adds armor and tempt range
+	armor: generic armor, armor
```

Comment:
does this allow `armour`?

Reply:
this is just for 1.21.3, so this is out of scope